sdk-blueprints/python/api/templates/model_test.mustache

```diff
@@ -0,0 +1,65 @@
+# coding: utf-8
+
+{{>partial_header}}
+
+import unittest
+
+{{#models}}
+{{#model}}
+import {{modelPackage}}
+from {{modelPackage}}.{{classFilename}} import {{classname}}
+
+class Test{{classname}}(unittest.TestCase):
+    """{{classname}} unit test stubs"""
+
+    def setUp(self):
+        pass
+
+    def tearDown(self):
+        pass
+{{^isEnum}}
+
+    def make_instance(self, include_optional) -> {{classname}}:
+        """Test {{classname}}
+            include_optional is a boolean, when False only required
+            params are included, when True both required and
+            optional params are included """
+        # uncomment below to create an instance of `{{{classname}}}`
```

Comment:
left-over comment that should be removed?

Reply:
Yep, forgot to remove.